.github/workflows/git.yaml

```
@@ -0,0 +1,13 @@
+name: Git Check
+
+on: [pull_request]
+
+jobs:
+  block-fixup-merge:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2.0.0
```

```suggestion
      - uses: actions/checkout@v2
```

Just [for consistency](https://github.com/lmc-eu/cookie-consent-manager/pull/3/files#diff-4b122024a3a28ded65da76a2f1bface1f3a27328374438d1298d25585fe0603bR14) :)